docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common acronyms.
+
+## CustomResourceDefinitions
+
+### CatalogSources
+
+**Definition**: The CatalogSource is a repository of [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and packages that define an application. 
+**Acronyms**: CatSrc(s)
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion is a required resource for an Operator which includes application metadata such as name, description, version, repository link, labels, icon, etc. It also includes a list of corresponding/required resources and their statuses, install strategy of the Operator as [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), supported install modes, etc.
```

Comment:
I really like this definition of the CSV[0]. Of course it's a little too verbose for the OLM book, maybe it can be summarized for the OLM book?   
[0] https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/philosophy.md#clusterserviceversion

Reply:
I agree with @anik120 here. I think this definition lists what the CSV _has_, but it doesn't really say what the CSV is _for_. I really like the first sentence in Anik's link:

> Represents a particular version of the ClusterService and the operator managing it